frontend/src/customAxios.ts

```diff
@@ -33,60 +33,55 @@ export const CustomAxios = ({
   envelope?: typeof defaultEnvelope;
   baseURL?: string;
 } = {}) => {
-  const token = store.getState().jwt;
-  const loadingToastId = React.useRef<React.ReactText | undefined>();
+  let loadingToastId: React.ReactText | undefined = undefined;
 
-  const instance = React.useMemo(() => {
-    const instance = axios.create({
-      baseURL,
-      headers: {
-        'Access-Control-Allow-Origin': '*',
-      },
-    });
-    instance.interceptors.request.use((config) => {
-      config!.headers!.Authorization = `Bearer ${token}`;
-      if (selector !== undefined) {
-        const state = store.getState();
-        const storedValue = selector(state);
+  const instance = axios.create({
```

Comment:
I don't know if it matters, but anything that uses this object will get redrawn infinitely because this hook is returning an object that changes every time the hook gets called.

Reply:
@Fosol I did some digging and found that my commit on September 1st: https://github.com/bcgov/PIMS/commit/324a5962742e30eb0f4a65e890059f926a65cc9e had introduced this new "useMemo" hook, I guess because I have ESLint and format on save turned on. Your original "customAxios.ts" didn't have it and neither does the existing version in the dev branch.

Reply:
If it works then it should be okay.  Hooks can have some unexpected behaviour because of this.